static/js/actions/financial_aid.js

```diff
@@ -16,7 +16,9 @@ export const CLEAR_CALCULATOR_EDIT = 'CLEAR_CALCULATOR_EDIT';
 export const clearCalculatorEdit = createAction(CLEAR_CALCULATOR_EDIT);
 
 export const UPDATE_CALCULATOR_EDIT = 'UPDATE_CALCULATOR_EDIT';
-export const updateCalculatorEdit = createAction(UPDATE_CALCULATOR_EDIT);
+export const updateCalculatorEdit = createAction(UPDATE_CALCULATOR_EDIT, 
+  option => option.value || option
```

Comment:
Why was this change made? It seems like we could add a `option.value => value` function at the caller site instead to make this consistent. I haven't looked over most of the code yet though so I don't know how it affects code cleanliness elsewhere